espnet/nets/pytorch_backend/transducer/decoder_interface.py

```diff
@@ -0,0 +1,100 @@
+"""Transducer decoder interface module."""
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
+
+import torch
+
+
+class TransducerDecoderInterface:
```

Comment:
Is it possible to add this class to https://github.com/espnet/espnet/blob/master/espnet/nets/scorer_interface.py or moving this file as https://github.com/espnet/espnet/blob/master/espnet/nets/scorer_interface_transducer.py ?

@ShigekiKarita, what do you think?
I want to place it in the same manner as our output-sync scorer interface.

Reply:
I agree with scorer_interface_transducer.py or transducer_decoder_interface.py (the module name should be equal to the class name)

Reply:
If it's okay for you, I prefer changing to `transducer_decoder_interface.py` as it's not really the same interface as ScorerInterface and other ones.

Reply:
OK for now, but we'd like to unify them in the near future.
So, could you please leave some documents of how they are different and the possibility of unifying them?